Jenkinsfile

```diff
@@ -12,7 +12,14 @@ node(){
   }
 
   stage('Deploy')   {
-      cloudFoundryDeploy script:this, deployTool: 'mtaDeployPlugin'
+      def id = new Date().getTime()
+      consumerTestsSpace = "consr-test-${id}"
```

Comment:
how about `ct` for consumer test. **Or**, more often than not, we call them integration tests, so `it` would also make sense. As we are talking of a space, so environment, I would actually prefer something like `val` to show that this is a validation space. A compromise would be something like `val-it-${id}`